js/google_ima.js

```diff
@@ -1034,6 +1034,7 @@ require("../html5-common/js/utils/utils.js");
         //These are required by Google for tracking purposes.
         google.ima.settings.setPlayerVersion(PLUGIN_VERSION);
         google.ima.settings.setPlayerType(PLAYER_TYPE);
+        google.ima.settings.setLocale(OO.getLocale());
```

Comment:
getLocale(): `return (OO.locale || document.documentElement.lang || navigator.language || navigator.userLanguage || "en").substr(0,2).toUpperCase();`

The getLocale will default to `"EN"` if everything before it is falsey.